ansible/lm-helm-values.yml

```diff
@@ -134,6 +134,9 @@ filebeat:
     hostPath:
       path: /var/snap/microk8s/common/var/lib/docker/
 
+kibana:
+  enabled: true
+
```

Comment:
Note that the ansible_variables.yml has a variable that defines whether lm-allinone enables Kibana (see also https://github.com/accanto-systems/ansible-role-lifecyclemanager/blob/master/templates/base_values.yml, which uses this variable). Why are we overriding it to always be true here?

Reply:
Since the previous commit set logging dashboard to always be true, whenever an error window pops up there will be a link to "View in Kibana" and clicking on that link would lead to nowhere since Kibana is not installed. 
This way it would be less confusing for a first-time user.

Reply:
But it's still possible to override whether Kibana is installed by setting ansible_variables.yml "lm_kibana_enabled" variable to "False". If the logging dashboard is always on, then perhaps it makes sense to remove the "lm_kibana_enabled" variable and fix up the base_values.yml file to set kibana.enabled to "true"

Reply:
@sglover I reverted my earlier change and also removed `lm_kibana_enabled` variable on the role repo like you suggested https://github.com/accanto-systems/ansible-role-lifecyclemanager/pull/1 Can you review both pull requests?